packages/blockchain/CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 6.0.0-beta.3 - 2022-08-10
+
+Beta 3 release for the upcoming breaking release round on the [EthereumJS monorepo](https://github.com/ethereumjs/ethereumjs-monorepo) libraries, see the Beta 1 release notes for the main long change set description as well as the Beta 2 release notes for notes on some additional changes ([CHANGELOG](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/blockchain/CHANGELOG.md)).
+
+## Blockchain Interface Changes
+
+The Blockchain interface has been expanded by a few methods and is now guaranteed to work with the VM, following properties and methods have been added, see PR [#2069](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2069):
+
+- `consensus: Consensus` property (for the consensus implementation)
+- `copy(): BlockchainInterface` method
+- `validateHeader(header: BlockHeader, height?: bigint): Promise<void>` method
+
+The following methods are added, but are optional to implement:
+
+- `getIteratorHead?(name?: string): Promise<Block>` method
+- `getTotalDifficulty?(hash: Buffer, number?: bigint): Promise<bigint>` method
+- `genesisState?(): GenesisState` method
+- `getCanonicalHeadBlock?(): Promise<Block>` method
+
+## Other Changes
+
+- Update to renamed `hardforkByTTD` (before: `hardforkByTD`) option for Block instantiations, PR [#2075](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2075)
```

Comment:
Should we also mention that we've added the optional maxBlocks param to the `iterator` method?
See line 41 of `packages/blockchain/src/types.ts`: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2069/files#diff-2f38c58db4b3c2c6fb7a8f3b56f54717698a1715fb3b9b405fd046b202fb0fec

Reply:
Sorry, have to decline some comments, really low on time. I had considered this but left for time reasons.